runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingStepMetricsContainer.java

```diff
@@ -50,10 +57,9 @@
   "nullness" // TODO(https://github.com/apache/beam/issues/20497)
 })
 public class StreamingStepMetricsContainer implements MetricsContainer {
-
   private final String stepName;
 
-  private static Boolean enablePerWorkerMetrics;
+  private static Boolean enablePerWorkerMetrics = false;
```

Comment:
unless this is nullable, use the primitive type (`boolean`)

Reply:
Done